src/config/chain/chain.ts

```diff
@@ -49,3 +53,8 @@ export const resolveChainQuery = (symbol: string): Chain => {
   if (!chain) throw new BadRequest('Invalid chain');
   return chain;
 };
+
+export const initStrategies = (): void => {
```

Comment:
Should we make the strategies services so they are injectable? Can be done in the future if we move away from util files